system/services/admin/AuditService.cfc

```diff
@@ -32,6 +42,49 @@ component output="false" singleton=true {
 		} );
 	}
 
+	public query function getAuditLog(
+		  numeric startRow = 1
+		, numeric maxRows  = 0
+		) output=false {
+		var records = _getDao().selectData(
```

Comment:
Indentation is a little off here. Looks like two tabs when only one required. Also, closing `)` should align with the `var`, so:

```
var records = _getDao().selectData(
	  selectFields = [ "audit_log.id", "audit_log.type", "audit_log.datecreated", "audit_log.action", "security_user.email_address", "security_user.known_as" ]
	, orderby      = "audit_log.datecreated desc"
	, startRow     = arguments.startRow
	, maxRows      = arguments.maxRows
);
```